config/settings.yml

```diff
@@ -1,23 +1,23 @@
 multitenancy:
   enabled: false
-  default_host: "<%= ENV.fetch('SETTINGS__MULTITENANCY__DEFAULT_HOST', '%{tenant}.dev') %>"
+  default_host: "%{tenant}.dev"
```

Comment:
I disagree; I'm not sure there's much value in polluting our readme with implementation details about upstream gems (although a sentence saying "there's a bunch of settings in `config/settings.yml` seems reasonable, but I'd argue unrelated to this PR...).